assets/src/edit-story/elements/image/edit.js

```diff
@@ -46,20 +47,24 @@ const Element = styled.div`
 
 const FadedImg = styled.img`
   position: absolute;
-  opacity: 0.4;
+  opacity: ${({ opacity }) =>
+    opacity ? opacity * MEDIA_MASK_OPACITY : MEDIA_MASK_OPACITY};
   pointer-events: none;
   ${imageWithScale}
 `;
 
 const CropImg = styled.img`
   position: absolute;
-  ${imageWithScale}
+  opacity: ${({ opacity }) =>
+    opacity ? opacity - opacity * MEDIA_MASK_OPACITY : null};
+  ${imageWithScale};
```

Comment:
The semicolon here is not needed.

```suggestion
  ${imageWithScale}
```